mobile/lib/main.dart

```diff
@@ -549,8 +549,13 @@ Future<void> runBackend(bridge.Config config) async {
     appDir = seedDir;
   }
 
-  FirebaseMessaging messaging = FirebaseMessaging.instance;
-  final fcmToken = await messaging.getToken();
+  String fcmToken;
+  try {
+    fcmToken = await FirebaseMessaging.instance.getToken().then((value) => value ?? '');
+  } catch (e) {
+    FLog.error(text: "Error fetching FCM token: $e");
+    fcmToken = '';
```

Comment:
Given we send this token to the coordinator to register, should we set it here explicitly to something so that we know the user had an error and we could not get a token for them ?